sendgrid/helpers/exceptions/__init__.py

```diff
@@ -0,0 +1 @@
+from .exceptions import *
```

Comment:
Oh. I missed this. It is always a good idea to explicitly mention the imports. (Python Zen ;) )